### PR TITLE
docs(eks): added scaling in eks

### PR DESCRIPTION
**Pull request by brianosburn-intel**
closes [AB#3476](https://dev.azure.com/rbheBoards/c7d9d701-a85f-45ff-91f6-26c56ff2c4de/_workitems/edit/3476)

## PR Checklist

- [x ] Leverages formatting as provided by https://squidfunk.github.io/mkdocs-material/reference/abbreviations/ 
- [x ] Have viewed and verified edits in rendered form ensuring proper formatting

## What are you changing?

adding k8 support for Amazon AWS

### CODE REVIEW

**Comment by bwendlandt-intel** [comment]
@rsdmike Am I able to get access to our AWS instance to test this or are we limited on users?

File: docs/Kubernetes/deployingk8s-eks.md [hunk 1]
```diff
@@ -0,0 +1,353 @@
+--8<-- "References/abbreviations.md"
+
+# Amazon Elastic Kubernetes Service (EKS)
+
+This tutorial demonstrates how to deploy the Open AMT Cloud Toolkit on a Kubernetes cluster using EKS. Alternatively, you can also perform a simpler, test deployment using a single-node cluster locally. See [Kubernetes (K8s)](./deployingk8s.md).
```
Comment (kimberlyxdavis) [inline docs/Kubernetes/deployingk8s-eks.md:5]: The thing I might question here is phrasing this as "alternatively".  We're trying to get away from phrasing things as in "you can do it this way and that way and whatever way". It's confusing to the user. 

To make this sound like a separate thing, I might say: 
To perform a simpler test deployment, use a single-mode cluster locally. See [ ].

File: docs/Kubernetes/deployingk8s-eks.md [hunk 2]
```diff
@@ -0,0 +1,353 @@
...
+
+
+### 2. Create ssh key for cluster access
+
+Follow steps listed [here](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html#having-ec2-create-your-key-pair).
+
+
+### 3. Create cluster
+
+Perform the following command to create a new EKS cluster as well as supporting components.
```
Comment (kimberlyxdavis) [inline docs/Kubernetes/deployingk8s-eks.md:32]: Even shorter:

Create a new EKS cluster and supporting components:

File: docs/Kubernetes/deployingk8s-eks.md [hunk 3]
```diff
@@ -0,0 +1,353 @@
...
+
+
+## Create postgres db in RPS
+
+### 1. Create a managed instance of postres db
+[Creating Postgres DB steps](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_GettingStarted.CreatingConnecting.PostgreSQL.html#CHAP_GettingStarted.Creating.PostgreSQL)
+Note: Use the vpc created from the cluster above. Select the default security group.
+
+
+### 2. Configure VPC for access
```
Comment (kimberlyxdavis) [inline docs/Kubernetes/deployingk8s-eks.md:52]: What's the VPC again? Virtual Private Cloud?

File: docs/Kubernetes/deployingk8s-eks.md [hunk 4]
```diff
@@ -0,0 +1,353 @@
...
+
+1. go to [RDS home](https://console.aws.amazon.com/rds/home)
```
Comment (kimberlyxdavis) [inline docs/Kubernetes/deployingk8s-eks.md:54]: Why are these sentences lower case? Is there a open source convention reason?

File: docs/Kubernetes/deployingk8s-eks.md [hunk 5]
```diff
@@ -0,0 +1,353 @@
...
+
+### 3. Create cluster
+
+Perform the following command to create a new EKS cluster as well as supporting components.
+
+```
+eksctl create cluster --name <name> --region <region> --with-oidc --ssh-access --ssh-public-key <ssh keypair name> --managed
+```
+
+Where:
```
Comment (kimberlyxdavis) [inline docs/Kubernetes/deployingk8s-eks.md:38]: Parameters:

File: docs/Kubernetes/deployingk8s-eks.md [hunk 6]
```diff
@@ -0,0 +1,353 @@
...
+
+4. Run the following command to update the stack using helm
+
+```
+helm upgrade openamtcloudstack ./kubernetes/charts 
+```
+
+## Amazon EBS CSI driver
+
+Enable persistant storage in the cluster be following the steps provided in the following [guide](https://docs.aws.amazon.com/eks/latest/userguide/ebs-csi.html).
```
Comment (kimberlyxdavis) [inline docs/Kubernetes/deployingk8s-eks.md:327]: Spelling: persistent

File: docs/Scaling/overview.md [hunk 7]
```diff
@@ -52,3 +52,11 @@ AKS is a container orchestration system that enables administrators to deploy an
     The AKS deployment section is not a tutorial for beginners. It intended for those who have prior knowledge of the service. To begin learning about AKS, start with [Azure Kubernetes Service](https://docs.microsoft.com/en-us/azure/aks/).
 
 [Get Started with AKS](../Kubernetes/deployingk8s-aks.md){: .md-button .md-button--primary target="_blank"}
+
+## Amazon Elastic Kubernetes Service (EKS)
+EKS is a container orchestration system that enables administrators to deploy and manage large numbers of containers and services. 
+
+!!! Warning
+    The EKS deployment section is not a tutorial for beginners. It intended for those who have prior knowledge of the service. To begin learning about EKS, start with [Amazon Elastic Kubernetes Service](https://docs.aws.amazon.com/eks/latest/userguide/getting-started.html).
```
Comment (kimberlyxdavis) [inline docs/Scaling/overview.md:60]: It is intended

File: docs/Scaling/overview.md [hunk 8]
```diff
@@ -52,3 +52,11 @@ AKS is a container orchestration system that enables administrators to deploy an
...
+
+[Get Started with EKS](../Kubernetes/deployingk8s-eks.md){: .md-button .md-button--primary target="_blank"}
```
Comment (kimberlyxdavis) [inline docs/Scaling/overview.md:62]: Lastly, I would just say try to be consistent ending complete sentences with periods unless they are a heading and using colons appropriately.

**Review by kimberlyxdavis — CHANGES_REQUESTED** [review]

**Review by kimberlyxdavis — APPROVED** [review]

**Pull request merged**